lib/registration.pm

```diff
@@ -262,8 +262,15 @@ sub registration_bootloader_params {
 }
 
 sub yast_scc_registration {
-
-    type_string "yast2 scc; echo yast-scc-done-\$?- > /dev/$serialdev\n";
+    my ($type_slow) = @_;
+    my $yast_scc_cmd = "yast2 scc; echo yast-scc-done-\$?- > /dev/$serialdev\n";
+    #Due to ninja keys on PPC, may be required to type command slowlier, see poo#19230
+    if ($type_slow) {
```

Comment:
Yes, tests on malbec are much more stable. Exactly this line failed couple of times due to redirection symbol, which requires shift key, other strings seem to work fine. This is same case when alt key is pressed and sometimes not released before. In general this change has to be reverted once it is fixed in framework, as I believe it should work even if worker is slow.